src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/food/HeavyCream.java

```diff
@@ -0,0 +1,28 @@
+package io.github.thebusybiscuit.slimefun4.implementation.items.food;
+
+import io.github.thebusybiscuit.slimefun4.api.events.PlayerRightClickEvent;
+import io.github.thebusybiscuit.slimefun4.core.handlers.ItemUseHandler;
+import me.mrCookieSlime.Slimefun.Lists.RecipeType;
+import me.mrCookieSlime.Slimefun.Objects.Category;
+import me.mrCookieSlime.Slimefun.Objects.SlimefunItem.SlimefunItem;
+import me.mrCookieSlime.Slimefun.api.SlimefunItemStack;
+import org.bukkit.inventory.ItemStack;
+
+/**
+ * This {@link SlimefunItem} can be obtained by crafting and, it's
+ * used for various food recipes recipes
+ *
+ * @author TheSilentPro
+ */
+public class HeavyCream extends SlimefunItem {
+
```

Comment:
ParamsAreNonnullByDefault

Reply:
Add it to the class?
May you tell me what that does, haven't really seen it in other sf classes?

Reply:
To the constructor, it marks the parameters to be "non-null".
It is used for most of our constructors and methods.

I see the preview being a bit inaccurate there...
I marked the line above the constructor but forgot that it doesn't show the constructor below in the preview :/